app/controllers/shipit/api/ccmenu_controller.rb

```diff
@@ -0,0 +1,33 @@
+module Shipit
+  module Api
+    class CcmenuController < BaseController
```

Comment:
I think you can register `CCMenu` as acronym, which means you can capitalize it: http://api.rubyonrails.org/classes/ActiveSupport/Inflector/Inflections.html#method-i-acronym

There's usually an initializer (in `config/initializers`) that sets acronyms, irregular plurals and such,